manifest.json

```diff
@@ -1,7 +1,7 @@
 {
   "vendor": "vtex",
   "name": "b2b-quotes",
-  "version": "1.5.5",
+  "version": "1.5.6",
```

Comment:
This shouldn't be changed here in the PR. It will be changed automatically by the vtex io bot once this PR is merged. You should revert this change, and just select the right type of version bump in the question post from vtex io bot.